File: week1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int h;

    do
    {
        h = get_int("Height: ");
    }
    while (h < 1 || h > 8);

    for (int i = 0; i < h; i++)
    {
        printf("%.*s", h - i - 1, "        ");
        printf("%.*s", i + 1, "########");
        printf("%.*s", 2, "  ");
        printf("%.*s\n", i + 1, "########");
    }
}

File: week6/sentimental-mario-more/mario.py
from cs50 import get_int


while True:

    h = get_int("height: ")

    if h > 0 and h <= 8:
        break
    else:
        pass

maxh = h


def ppyramid(height):

    if (height <= 0):
        return

    ppyramid(height-1)

    print(' ' * (maxh - height) + '#' * height + '  ' + '#' * height)


ppyramid(h)
